ocaml/xapi/cli_operations.ml

```diff
@@ -3195,7 +3195,8 @@ let vm_import fd printer rpc session_id params =
 														* alert it to the failure. *)
 
 														(let l=Client.Task.get_current_operations rpc session_id importtask in
-															if List.exists (fun (_,x) -> x=`cancel) l then raise (Api_errors.Server_error(Api_errors.task_cancelled,[])));
+															if List.exists (fun (_,x) -> x=`cancel) l
```

Comment:
Grief, I thought this was just blank lines until I scrolled over to the right. This function could do with some refactoring, methinks.